package.json

```diff
@@ -114,6 +114,7 @@
     "eslint-config-airbnb": "^15.1.0",
     "eslint-config-prettier": "^3.1.0",
     "eslint-import-resolver-babel-module": "^4.0.0",
+    "eslint-plugin-enzyme-always-unmount": "./script/custom-eslint/enzyme-always-unmount",
```

Comment:
Is this a thing specific to eslint? It looks cool, but I haven't seen this before.

Reply:
Looks like it's described here, as the very last option for setting the source of a dependency https://docs.npmjs.com/files/package.json#dependencies

Reply:
I see local paths, but they're prefixed by `file:`. Maybe I'm reading it wrong 🤷‍♂️

Reply:
Oh yeah... you're right. Weird... But this seems to work? 🤔

Reply:
:point_up:

Reply:
yarn did that so maybe it's wrong. 
![screen shot 2018-12-06 at 15 02 46](https://user-images.githubusercontent.com/4998130/49614608-032d1000-f968-11e8-888a-ffe2d20c281e.png)